### PR TITLE
Layout and footer

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

**Comment by dzucconi** [comment]
The whole layout component needs a bit more re-organization to make sense and I've left some `TODO`s but it's an OK start.

File: components/layout/index.jade [hunk 1]
```diff
@@ -1,53 +1,41 @@
...
 
-  body(class=extraClasses)
+  body( class= extraClasses )
+    //- TODO: Refactor to use modalize, only thing that uses this is AuthView
     #modal-container
       //- Rendered client-side
 
-    #l-lightbox-container
-      //- Rendered client-side
-
```
Comment (dzucconi) [inline components/layout/index.jade]: I'm 99% sure nothing uses this. The only thing I can find is part of the block view ganks some styles from that component.

Comment (dzucconi) [inline components/layout/index.jade]: cc @broskoski

File: components/layout/index--infinite.jade [hunk 2]
```diff
@@ -0,0 +1,41 @@
+- colorClass = sd.IS_INVERTED ? 'is-inverted' : ''
+- loggedInClass = user ? 'is-logged-in' : 'not-logged-in'
+- extraClasses = ''
```
Comment (dzucconi) [inline components/layout/index--infinite.jade]: I can't really refactor these out for now but whenever you include *any* variables inline within a template, ensure they are at the very top.

File: components/layout/index.jade (hunk 1 above)
Comment (broskoski) [inline components/layout/index.jade]: Yeah, I think we can remove this.

**Comment by broskoski** [comment]
So this will actually add the footer to the current setup, and you're going to add the "politeness" later?

**Comment by dzucconi** [comment]
Infinite scroll stuff lacks footer. Everything else has it.

**Comment by dzucconi** [comment]
"Politeness" is a much larger and more complicated issue and should be spec'd and opened separately.

**Comment by broskoski** [comment]
👍

**Pull request merged**